### PR TITLE
Add video and apk file types

### PR DESCRIPTION
**Pull request by devslava**

### CODE REVIEW

File: library/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -11,7 +11,9 @@
     <string name="type_excel">Spreadsheet</string>
     <string name="type_image">Image</string>
     <string name="type_music">Music</string>
+    <string name="type_video">Видео</string>
```
Comment (lukaville) [inline library/src/main/res/values/strings.xml]: Use english language in this file (write "Video").

Comment (devslava) [inline library/src/main/res/values/strings.xml]: It's my mistake:)

Comment (devslava) [inline library/src/main/res/values/strings.xml]: I've corrected alredy. :)

**Comment by lukaville** [comment]
Thanks!

**Pull request merged**